content/docs/introduction/extra-usage.md

```diff
@@ -0,0 +1,158 @@
+- **Unit Price** is the cost per unit
+- **Days** is the total number of days in the month
+- **Days Left in Month** is the number of days remaining in the month after going over your limit
+
+Once an extra unit of storage or projects is allocated, you are billed for that extra unit for the remainder of the month. If you reduce your usage during that month and no longer require extra units of storage or projects, the extra usage charge is dropped at the beginning of the next month when your bill resets based on current usage.
+</Admonition>
+
+**Compute**
+
+Extra compute usage is available with the [Launch](/docs/introduction/plans##launch) and [Scale](/docs/introduction/plans##scale) plans and is billed by _compute hour_ at $0.16 per hour. For example, the Launch plan has an allowance of 300 compute hours included in the plan's monthly fee. If you use 100 additional compute hours over the billing period, you are billed an extra $16 (100 x $0.16). Since extra compute usage is per hour, prorated billing does not apply.
+
+## Extra usage examples
+
+The following examples illustrate how extra storage is allocated and billed in Neon. The same method applies to extra project usage. The examples are based on the Scale plan, which comes with a 50 GiB storage allowance. The Launch plan has a different storage allowance (10 GiB) and cost per extra storage unit ($3.50 per 2 GiB), but the examples still apply.
+
+### Example 1: Steady extra storage usage
+
+This example illustrates how a steady amount of extra storage usage is billed.
+
+**Plan**: Scale
+**Period:** June 1st – June 30th  
+**Usage:** Steady at 55 GiB throughout the month
+
+| Item          | Details                            |
+|---------------|------------------------------------|
+| Plan Fee      | 1 month = $69                                |
+| Extra Storage | 10 GiB extra storage pack at $15/month from June 1st – June 30th = $15    |
+| **Total**| $84                 |
+
+### Example 2: Storage exceeds the limit at the start of the month
+
+This example illustrates how extra storage is billed from the date the extra storage usage is allocated until the end of the month, not just for the days the storage limit was exceeded.
+
+**Plan**: Scale
+**Period:** June 1st – June 15th  
+**Usage:** 55 GiB  
+**Change:** On June 16th, usage decreased to 45 GiB until the end of the month
+
+| Item          | Details                            |
+|---------------|------------------------------------|
+| Plan Fee      | 1 month = $69                                |
+| Extra Storage | 10 GiB extra storage pack at $15/month from June 1st – June 30th = $15    |
+| **Total**| $84                 |
+
+<Admonition type="note">
+If usage remained at 45 GiB through to the end of July, no extra storage would be needed, and July's total would be $69 for the Scale plan.
+</Admonition> 
+
+### Example 3: Storage exceeds and falls below the limit at the end of the month
```

Comment:
I wonder if less formal "spike" might make the scenario easier to understand at a glance?

"Storage spikes briefly at the end of month"

Reply:
changed as suggested